appveyor.yml

```diff
@@ -39,7 +38,7 @@ for:
     - ps: . "$env:SCOOP_HOME\test\bin\test.ps1" -TestPath "$env:APPVEYOR_BUILD_FOLDER"
 - matrix:
     only:
-      - PowerShell: 6
+        - PowerShell: Current
```

Comment:
Same as above.

Reply:
I prefer to know as soon as a major version breaks and not rely on user reports, but I have made the change you requested.